packages/next-on-pages/templates/_worker.js/routes-matcher.ts

```diff
@@ -415,13 +415,16 @@ export class RoutesMatcher {
 		}
 
 		const isLocaleIndex =
-			/^\//.test(route.src) && route.src.slice(1) in this.locales;
+			/^\/$/.test(route.src) && route.src.slice(1) in this.locales;
```

Comment:
Is this not the same as:

```suggestion
			route.src === "/" && "/" in this.locales;
```

Reply:
yes thanks 🙂, I missed this when updating the regex

Although `"/" in this.locales` is always going to return false since locales contains the actual locales (not paths)

So I might just completely remove this check I think 😕 🤔

Reply:
Ah no ok, this is something I already looked into and tweaked before (we were adding `^` and `$` to specific cases here, but that's no longer needed as we now do it for all routes)

This code was already a noop but I forgot to remove it, thanks a lot for pointing it out! 🙏